src/Files.App/Data/Models/ColumnsViewModel.cs

```diff
@@ -141,7 +129,7 @@ public override bool Equals(object? obj)
 
 			if (obj is ColumnsViewModel model)
 			{
-				return (
+				return
```

Comment:
I think this is technically a single line if so we could drop the brackets

Reply:
That doesn’t reduce readability?

Reply:
Ok, sounds good